neqo-crypto/src/result.rs

```diff
@@ -34,16 +25,12 @@ where
     }
 }
 
-fn result_helper(rv: ssl::SECStatus, allow_blocked: bool) -> Res<bool> {
+pub fn result(rv: ssl::SECStatus) -> Res<()> {
```

Comment:
Is there a way to make places where we're calling `result::result` a little more clear? Can it be named `res_from_status` or `res_from_rv` or something similar?

Reply:
or can an `impl TryFrom<ssl::SECStatus> for Res` be of help?

Reply:
Given that `SECStatus` is always convertable, that would be `From`, wouldn't it?  Except that we don't define any of the involved types:
* `SECStatus` is a type alias for `i32`.
* `Res` is a type alias for `Result`.
* `From` is similarly in the standard library.

This seems like a great idea, but we can't realistically do that.  Renaming the function is the best we can do.  `res_from_secstatus` is the best I can come up with offhand.  I've opened #166.